cmd/minishift/cmd/config/validations.go

```diff
@@ -41,7 +41,7 @@ func IsValidDriver(string, driver string) error {
 }
 
 func RequiresRestartMsg(name string, value string) error {
-	api := libmachine.NewClient(constants.Minipath, constants.MakeMiniPath("certs"))
+	api := libmachine.NewClient(constants.GetMinipath(), constants.MakeMiniPath("certs"))
```

Comment:
Well, I am pretty sure that if we wanted to we could keep constants.Minipath. There is no reason why this should not work. If we, however, move this out of constant.go we will have the same amount of code change again, since in this case at least _constant._ would change. 

Personally, I probably would try with constants.Minipath to start with, keeping the change small initially. Once everything would be working, one can apply the name refactoring as a separate step.